driver-core/src/main/java/com/datastax/driver/core/PreparedStatement.java

```diff
@@ -125,7 +125,7 @@
    * Sets a default consistency level for all bound statements created from this prepared statement.
    *
    * <p>If no consistency level is set through this method, the bound statement created from this
-   * object will use the default consistency level (ONE).
+   * object will use the default consistency level (LOCAL_ONE).
```

Comment:
Not related to this PR, but it was wrong. We'll keep the commit separate.

Reply:
Good catch